src/workers/parser/getSymbols.js

```diff
@@ -159,10 +160,12 @@ function extractSymbols(source: Source) {
 
       if (t.isCallExpression(path)) {
         const callee = path.node.callee;
+        const args = path.node.arguments;
         if (!t.isMemberExpression(callee)) {
           const { start, end, identifierName } = callee.loc;
           callExpressions.push({
             name: identifierName,
+            values: args.filter(arg => arg.value).map(arg => arg.value),
```

Comment:
Is just storing values better? Also, I use a filter first to trim the null / undefined values

Reply:
this looks good to me